content/inside-rust/call-for-testing-hint-mostly-unused.md

```diff
@@ -0,0 +1,207 @@
++++
+path = "inside-rust/2025/07/14/call-for-testing-hint-mostly-unused"
+title = "Call for Testing: Speeding up compilation with `hint-mostly-unused`"
+authors = ["Josh Triplett"]
++++
+
+I'm pleased to announce, and call for testing of, the nightly-only `rustc`
+`-Zhint-mostly-unused` option, and the corresponding nightly Cargo features
+`profile.hint-mostly-unused` and `hints.mostly-unused`. These options can help
+accelerate your Rust compile time in some cases, by avoiding compilation of
+items from your dependencies that you aren't using. Your feedback will help
+evaluate these features and make progress towards stabilizing them in the
+future.
+
+## Background
+
+When building a Rust library crate, the compiler generates compiled code for as
+much of the crate as it can (everything that isn't generic and isn't inlined),
```

Comment:
```suggestion
much of the crate as it can (everything that isn't generic and doesn't have `#[inline]`),
```

We should avoid using the adjective "inlined" for functions that have the inline attribute, as that adjective is more commonly used for functions that *actually have been inlined by the optimizer*. Let's avoid this confusion and be very explicit about the attribute.